docs/guides/vault/restrict-addresses-to-vaults.md

```diff
@@ -210,29 +190,26 @@ const allowedCallsDataValue = encodeKey(allowedCallsSchema, [
   vaultAddress,
   '0xffffffff',
```

Comment:
We need to update this to latest allowedCalls spec (Just a reminder for later when we update erc725.js)

Reply:
let's do this in a separate ticket @YamenMerhi @b00ste ?